### ISSUE TITLE
Не работает с OpenCart 1.5.6.4

### ISSUE THREAD

**Issue by ilerok17** [ulogin/ulogin-OpenCart#8]
При установке на 1.5.6.4 в админке выдает Fatal error: Call to undefined method Loader::controller() in /home/11111/data/www/****.com.ua/admin/controller/module/ulogin_sets.php on line 89

Почему? Есть ли модуль для 1.5.6.4?

**Comment by ipshenicyn** [comment]
Нет. Как вы можете увидеть в информации о плагине, он предназначен для версий 2+

**Comment by ilerok17** [comment]
Для 1.5 не планируется адаптация?:(

**Comment by ipshenicyn** [comment]
Конечно нет. Сколько уже этой версии? 3-4 года? Не видим смысла писать модуль для настолько устаревшей CMS. Мы бы, разумеется, пошли на это, если бы увидели интерес у вебмастеров. Но вы первый, на моей памяти, обратились за плагином под эту версию.
